### ISSUE TITLE
Harbor appliance - with https - fails with PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)

### ISSUE THREAD

**Issue by mobla** [vmware/harbor#994]
Harbor Build: 0.4.5

Steps:
1. Follow https://github.com/vmware/harbor/blob/master/docs/configure_https.md
2. Follow https://github.com/vmware/harbor/blob/master/docs/installation_guide_ova.md to use SSL Cert: & SSL Cert: Key
3. Boot the Harbor appliance
4. It does change the config to https but fails to decode the crt

Nov  2 05:06:23 172.18.0.1 docker/proxy[849]: 2016/11/02 05:06:23 [emerg] 1#1: PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)
Nov  2 05:06:23 172.18.0.1 docker/proxy[849]: nginx: [emerg] PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)
Nov  2 05:06:23 172.18.0.1 docker/proxy[849]: 2016/11/02 05:06:23 [emerg] 1#1: PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)
Nov  2 05:06:23 172.18.0.1 docker/proxy[849]: nginx: [emerg] PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)
Nov  2 05:06:24 172.18.0.1 docker/proxy[849]: 2016/11/02 05:06:24 [emerg] 1#1: PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)
Nov  2 05:06:24 172.18.0.1 docker/proxy[849]: nginx: [emerg] PEM_read_bio_X509_AUX("/etc/nginx/cert/server.crt") failed (SSL: error:0906D064:PEM routines:PEM_read_bio:bad base64 decode)

**Comment by mobla** [comment]
[logs.tar.gz](https://github.com/vmware/harbor/files/565729/logs.tar.gz)
[config.tar.gz](https://github.com/vmware/harbor/files/565728/config.tar.gz)

**Comment by reasonerjt** [comment]
The root cause is that the string of cert was updated ( all \n were removed) when you paste to vsphere client and passed to VM.
We used a script to add \n to header and footer, but the decoding still failed.

A newer version of openssl can decode the cert as expected so the nginx image will be bumped up to 1.11.5 to include the newer openssl library. 
A PR has been merged to master branch and .ova has been updated.
Thanks for reporting the issue.
